.github/workflows/comparison_results.yml

```diff
@@ -84,7 +84,7 @@ jobs:
 
       # langauge specific repository file
       - name: Run the script for ${{ env.HEAD_REF  }} and ${{ env.BASE_REF   }}
-        run: cd ./temp/standalone-monitoring-stability && pip install -r requirements.txt && python3 ./run.py -r ./repos/${{matrix.language}}.txt -b main -h main -guf --custom-joern True --custom-joern-base-branch ${{ env.BASE_REF }} --custom-joern-head-branch ${{ env.HEAD_REF }}
+        run: cd ./temp/standalone-monitoring-stability && pip install -r requirements.txt && python3 ./run.py -r ./repos/${{matrix.language}}.txt -b main -h dev -guf --custom-joern True --custom-joern-base-branch ${{ env.BASE_REF }} --custom-joern-head-branch ${{ env.HEAD_REF }}
```

Comment:
@ankit-privado Why are we doing this? It might be difficult to debug where the difference comes from, using the dev branch or the joern version.

Reply:
Yes right but the expected behaviour would be to pick dev branch by default. This is the same thing we are did for Privado-core also.

Reply:
I suggest using main for both comparisons. @pandurangpatil, what do you think?